clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -4280,7 +4280,7 @@ unsigned TokenAnnotator::splitPenalty(const AnnotatedLine &Line,
     //   aaaaaaa
     //       .aaaaaaaaa.bbbbbbbb(cccccccc);
     return !Right.NextOperator || !Right.NextOperator->Previous->closesScope()
-               ? 150
+               ? Style.PenaltyBreakBeforeMemberAccess
                : 35;
```

Comment:
I'm not sure if anything should be done about this `35`. I briefly considered using `Style.PenaltyBreakBeforeMemberAccess - 115`.